docs/developing-installer.md

```diff
@@ -0,0 +1,16 @@
+# Developing kni-installer
+
+In the normal workflow, dev-scripts extracts the installer binary out of
+a [release image](release-payload.md).  If you would like to develop and
+test changes to kni-install locally, set the `INSTALL_FROM_GIT`
+environment variable:
+
+```console
+$ INSTALL_FROM_GIT=true make
```

Comment:
a nit, but can we call this `KNI_INSTALLER_FROM_GIT` or something similar to make it clear this is something related to kni-installer?

Reply:
Renamed